articles/quickstart/native/ios-swift/04-calling-apis.md

```diff
@@ -15,46 +15,27 @@ budicon: 546
   ]
 }) %>
 
-The reason for implementing authentication, in the first place, is to protect information. In this case, your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. We recommend that you conform to RFC standards by sending valid authentication tokens through an authorization header.
+Auth0 provides a set of tools for protecting your resources with end-to-end authentication in your application. 
 
-In this tutorial, you'll learn how to get a token, attach it to a request (using the authorization header), and call any API you need to authenticate with.
+This tutorial shows you how to get a token, attach it to a request with an authorization header and call an API. We recommend you use this method for the best security and compliance with RFC standards. 
 
-## Get the user's credentials
+## Get the User's Credentials
 
-In order to make an authenticated request, you first need to obtain a token, against which your API can compare to detect whether or not the request is properly authenticated.
+You need a token for your API to check if the request is authenticated. 
```

Comment:
You need a token for your API to check if the request is authenticated.  -> You need an access token to authenticate your API request.